gooddata-java-model/src/main/java/com/gooddata/sdk/model/executeafm/afm/ComparisonCondition.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (C) 2019, GoodData(R) Corporation. All rights reserved.
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE.txt file in the root directory of this source tree.
+ */
+package com.gooddata.sdk.model.executeafm.afm;
```

Comment:
consider to move this to separate package as well as other conditions - it can make code more legible